docs/admin-guide/README.md

```diff
@@ -0,0 +1,3 @@
+# Administration Guide
+
+The following articals
```

Comment:
@kevgliss are you planning to complete this README?

Reply:
Will do.